sphinx/themes/basic/static/doctools.js

```diff
@@ -2,322 +2,182 @@
  * doctools.js
  * ~~~~~~~~~~~
  *
- * Sphinx JavaScript utilities for all documentation.
+ * Base JavaScript utilities for all Sphinx HTML documentation.
  *
  * :copyright: Copyright 2007-2021 by the Sphinx team, see AUTHORS.
  * :license: BSD, see LICENSE for details.
  *
  */
+"use strict"
 
-/**
- * select a different prefix for underscore
- */
-$u = _.noConflict();
-
-/**
- * make the code below compatible with browsers without
- * an installed firebug like debugger
-if (!window.console || !console.firebug) {
-  var names = ["log", "debug", "info", "warn", "error", "assert", "dir",
-    "dirxml", "group", "groupEnd", "time", "timeEnd", "count", "trace",
-    "profile", "profileEnd"];
-  window.console = {};
-  for (var i = 0; i < names.length; ++i)
-    window.console[names[i]] = function() {};
+const _ready = callback => {
```

Comment:
Suggestion: Format this code with [prettier](https://prettier.io/).

It makes a fairly reasonable stylistic choices, and notably, removes a large chunk of potential friction about code formatting.

Reply:
In c3a63dc8 I ran prettier with some options -- would it be better to run again with all defaults? (for reference I ran `prettier --print-width 120 --no-semi --quote-props as-needed --no-bracket-spacing --arrow-parens avoid --write sphinx/themes/basic/static`

A

Reply:
Just the defaults would be nicer for sure -- I'm extremely used to the default prettier style, TBH. 🙈

Reply:
Done in [9d9fa72](https://github.com/sphinx-doc/sphinx/pull/10028/commits/9d9fa72c4c1ed233af6d0d5d15691b430f91e534)